content/swift/concepts/sets/terms/remove.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '.remove()'
+Description: 'Removes an element at a specified index.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Sets'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.remove()`** method removes an element at a specified index.
+
+## Syntax
+
+```pseudo
+setName.remove(at: index)
+```
+
```

Comment:
Should we add a sentence under here to say something about how `index` should be within the bounds of `setName`?

Reply:
I didn't understand what you were saying. Can you explain it a little more?

Reply:
Should we include a sentence here that says that if the `index` argument passed to `.remove()` is greater than the length of `setName` minus 1, the `.remove()` method may throw an error?

Reply:
Sorry, I completely misunderstood what `.remove()` was really doing. I'm actually now wondering if `index` is the best parameter name? Based on your example below, wouldn't this be `value`?

Reply:
If you look at [the official document](https://developer.apple.com/documentation/swift/array/remove(at:)-1p2pj), you can see that the parameter must contain an index. Is there a reason why we should name it value?

Reply:
Hey @rriver2 it looks like you linked to the official docs for the `.remove()` method for _arrays_ but the issue you were assigned to pertains to the `.remove()` method for _sets_.

See here: https://developer.apple.com/documentation/swift/set/remove(_:)-8p2tv

Reply:
Based on this, we could call it either `value` or `element` but I think `index` makes more sense for an entry pertaining to arrays, not sets. (Codecademy Docs already seems to have [an existing `.remove()` array method entry](https://www.codecademy.com/resources/docs/swift/arrays/remove))